src/membraneWebRTC.ts

```diff
@@ -1372,6 +1379,22 @@ export class MembraneWebRTC {
     };
   };
 
+  private onIceCandidateError = (event: Event) => {
+    this.callbacks.onConnectionError?.("Ice candidate error occurred");
```

Comment:
This callback actually gets a more specific event - [`RTCPeerConnectionIceErrorEvent`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnectionIceErrorEvent)

It's not a critical error (i.e. it is possible to connect via a different ICE candidate), so I'd print it as a warning, but triggering `onConnectionError` is an overkill.

To give you more context, a real-life scenario where this occurs is TURN with an invalid TLS certificate. You put 3 entries in `iceServers`, the browser tries to connect 3 times using different protocols and UDP & TCP succeed, but TLS fails. In that scenario, the peer will connect unless his network restricts the network traffic to only allow TLS (rare case, but present in corporational networks). So, we only want to trigger `onConnectionError` when the peer fails to connect and it will be great to have that console log to debug the problem.

```suggestion
  private onIceCandidateError = (event: RTCPeerConnectionIceErrorEvent) => {
    console.warn(event);
```